implementations/go/README.md

```diff
@@ -1,5 +1,16 @@
 # Soccergist - Go
```

Comment:
This pull request is for java, why are there changes in the implementations/go directory?

Reply:
Did a lil work on the read me on the java branch ni. My Bad.